core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorter.java

```diff
@@ -522,7 +522,7 @@ public long spill() throws IOException {
           // is accessing the current record. We free this page in that caller's next loadNext()
```

Comment:
@rxin - We already have comment describing the spilling strategy here. Do you think we need more details?